### PR TITLE
add makensis

### PR DESCRIPTION
**Pull request by idleberg**
## Description

Talk about your changes here.

Fixes issue #

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [ ] User-visible changes noted in CHANGELOG.md

### CODE REVIEW

File: makensis.fish [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# System to create Windows installers
+# See: https://nsis.sourceforce.io
+
+complete -c makensis -o CMDHELP -d 'Prints out help for a specific command, or lists all commands'
```
Comment (faho) [inline makensis.fish]: So this is used as `makensis -CMDHELP`, with one "-"?

File: makensis.fish [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+complete -c makensis -o P5 -d 'Sets the compiler process priority to realtime'
+complete -c makensis -o PAUSE -d 'Pauses after execution'
+complete -c makensis -o PPO -d 'Preprocess to stdout/file'
+complete -c makensis -o SAFEPPO -d 'Preprocess to stdout/file'
+complete -c makensis -o V0 -d 'Verbosity: none'
+complete -c makensis -o V1 -d 'Verbosity: no warnings'
+complete -c makensis -o V2 -d 'Verbosity: no info'
+complete -c makensis -o V3 -d 'Verbosity: no script'
+complete -c makensis -o V4 -d 'Verbosity: all'
+complete -c makensis -o V4 -d 'Verbosity: all'
```
Comment (faho) [inline makensis.fish]: This line is duplicated.

File: makensis.fish [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+complete -c makensis -o OUTPUTCHARSET -a 'ACP OEM UTF8 UTF8SIG UTF16LE UTF16BE UTF16LEBOM UTF16BEBOM' -d 'Specifies output charset'
+complete -c makensis -o LICENSE -d 'Prints the makensis software license'
+complete -c makensis -o NOCD -d 'Disables the current directory change to that of the .nsi file'
+complete -c makensis -o NOCONFIG -d 'Disables inclusion of <path to makensis.exe>/nsisconf.nsh'
+complete -c makensis -o P0 -d 'Sets the compiler process priority to idle'
+complete -c makensis -o P1 -d 'Sets the compiler process priority to below normal'
+complete -c makensis -o P2 -d 'Sets the compiler process priority to normal'
+complete -c makensis -o P3 -d 'Sets the compiler process priority to above normal'
+complete -c makensis -o P4 -d 'Sets the compiler process priority to high'
+complete -c makensis -o P5 -d 'Sets the compiler process priority to realtime'
```
Comment (faho) [inline makensis.fish]: Below you have "Verbosity: all", why not say "compiler process priority: realtime" here?

File: makensis.fish [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# System to create Windows installers
+# See: https://nsis.sourceforce.io
```
Comment (ridiculousfish) [inline makensis.fish]: This link seems broken to me?

**Comment by faho** [comment]
Merged, thanks!

**Comment by mqudsi** [comment]
Am I missing something? Why is `makensis.fish` in the root?

**Comment by faho** [comment]
@mqudsi: Terribly sorry, I did not expect that!

**Comment by mqudsi** [comment]
No harm done :)

**Pull request merged**